libraries/AC_AttitudeControl/AC_AttitudeControl.cpp

```diff
@@ -1240,3 +1242,52 @@ void AC_AttitudeControl::get_rpy_srate(float &roll_srate, float &pitch_srate, fl
     pitch_srate = get_rate_pitch_pid().get_pid_info().slew_rate;
     yaw_srate = get_rate_yaw_pid().get_pid_info().slew_rate;
 }
+
+#if HAL_LOGGING_ENABLED
+// Write a rate packet
+void AC_AttitudeControl::Write_Rate(const AP_Motors &motors, const AC_PosControl &pos_control) const
```

Comment:
`AC_AttitudeControl` already has a local `_motors` reference. No need to pass it in.

Reply:
nice catch, fixed